### PR TITLE
Remove python 37

### PR DESCRIPTION
**Pull request by elboy3**
https://app.shortcut.com/galileo/story/6981/dq-remove-support-for-python-3-7

Additional context: https://galileoteamworkspace.slack.com/archives/C02NSBN4ZK3/p1690213086742379

From https://www.python.org/downloads/release/python-3717/
```
Note: The release you are looking at is Python 3.7.17, the final security bugfix release for the legacy 3.7 series which has now reached end-of-life and is no longer supported. See the downloads page for currently supported versions of Python.
```

### CODE REVIEW

**Comment by elboy3** [comment]
Will hold off merging until 8/8/23 after @setu4993 has had a chance to review :)

**Review by bogdan-galileo — APPROVED** [review]
thanks!!

File: dataquality/__init__.py [hunk 1]
```diff
@@ -151,8 +151,7 @@
 if sys.version_info < (3, 8):
     warn(
         "You are using an old version of Python. Please upgrade to Python 3.8"
-        "or higher. dataquality will stop supporting Python 3.7 in the near "
-        "future.",
+        "or higher. dataquality does not support Python 3.7 ",
         GalileoWarning,
     )
```
[the remark this answers is not in the document]
Comment (setu4993) [reply dataquality/__init__.py:156]: Actually, we can just delete L151-L156 since it can't even be installed on py37 or lower.

**Review by setu4993 — COMMENTED** [review]
🚀

Let's remove the block for the warning since we'd never hit it.

**Comment by elboy3** [comment]
@setu4993 👍 done

File: dataquality/__init__.py [hunk 2]
```diff
@@ -31,15 +31,13 @@
 """
 
 
-__version__ = "0.10.3"
+__version__ = "0.11.0"
```
Comment (setu4993) [inline dataquality/__init__.py:34]: I really would like us to start using true SemVer ~soon and avoid breaking changes in minor version bumps.

**Review by setu4993 — COMMENTED** [review]
🔥

**Review by setu4993 — APPROVED** [review]
🔥

**Pull request merged**